lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -51,7 +51,18 @@ defmodule GenServerTest do
   end
 
   test "call/3 exit messages" do
+    {:ok, pid} = GenServer.start_link(Stack, [:hello])
+
+    {:ok, stoped_pid} = GenServer.start_link(Stack, [:hello])
+    GenServer.stop(stoped_pid)
+
+    {:ok, killed_pid} = GenServer.start_link(Stack, [:hello], name: :stack)
+    GenServer.stop(:stack, :kill)
```

Comment:
You should use `GenServer.start` instead of start_link, otherwise the processes are linked and killing the GenServer will cause the test process to crash too!